rgw/v2/lib/rgw_config_opts.py

```diff
@@ -141,7 +141,7 @@ def set_to_ceph_cli(self, key, value, set_to_all=False, remote_ssh_con=None):
         out = json.loads(out_ps)
         daemon_name_list = []
         for node in out:
-            daemon_name = node.get("daemon_name")
+            daemon_name = node.get("service_name")
```

Comment:
we can use set_to_all=True, but that needs changes in all scripts where ever this is called for dynamic resharding..
this is kind of setting config to all daemons under that service in all scripts by default. I hope this will not cause any other issues.
if we have multiple rgw services, then set_to_all=True makes sense from now on..

LGTM